orm/src/test/java/org/hibernate/search/test/filter/FilterTest.java

```diff
@@ -160,6 +161,16 @@ public void testMultipleFiltersOfSameTypeWithDifferentParameters() {
 	}
 
 	@Test
+	public void testFilterDefinedOnSuperClass() {
+		TermQuery query = new TermQuery( new Term( "employer", "Red Hat" ) );
+		FullTextQuery ftQuery = fullTextSession.createFullTextQuery( query, Employee.class );
+		ftQuery.enableFullTextFilter( "roleFilter" )
+				.setParameter( "role", Role.ADMINISTRATOR );
+
+		assertEquals( "Should find the filter defined in the super class", 1, ftQuery.getResultSize() );
```

Comment:
I am not so font of this indirect testing. Really it should be enough to configure the filter and bootstrap Search. Afterwards one can just use `ExtendedSearchIntegrator#getFilterDefinition` to check whether the filter is there. Really that is what we want to confirm.